frontend/src/slices/auth/actions.ts

```diff
@@ -42,6 +42,17 @@ const signIn = createAsyncThunk<
   return user;
 });
 
+const signOut = createAsyncThunk<null, undefined, AsyncThunkConfig>(
+  `${sliceName}/sign-out`,
+  async (_, { extra, dispatch }) => {
+    const { storage } = extra;
+    await storage.drop(StorageKey.TOKEN);
+    dispatch(appActions.navigate(AppRoute.SIGN_IN));
```

Comment:
Does the private route already handle it?

Reply:
Are you saying that before adding the action I should have wrapped the sidebar in the protected route and inserted it in the index.tsx?

Reply:
As I know sign out functionality is available for auth user. Auth routes are wrapped with ProtectedRoute. That one has redirection depending on isAuthUser

Reply:
So basically, I do not need to explicitly define where to redirect the user since it will happen anyway? 
Okay, I do appreciate the clarification, thank you. 
I just wanted to make it visible what would happen in the end.

Reply:
user should be redirected to sign-in, but this should happen even without explicit redirect here